test/api/handlers_test.go

```diff
@@ -80,4 +81,51 @@ func testHandlerPinPOST(t *testing.T, at *test.Tester) {
 	if err != nil || status != http.StatusNoContent {
 		t.Fatal(status, err)
 	}
+
+	// Mark the skylink as unpinned and pin it again.
+	// Expect it to no longer be unpinned.
+	err = at.DB.SkylinkMarkUnpinned(at.Ctx, sl)
+	if err != nil {
+		t.Fatal(err)
+	}
+	status, err = at.PinPOST(sl)
+	if err != nil || status != http.StatusNoContent {
```

Comment:
why is `http.StatusNoContent` an acceptable response here?

Reply:
`StatusNoContent` (204) is `skyd`'s default success code and we're using it across all services as such.
`WriteSuccess()` writes a 204.

Reply:
no i get it now.